doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -448,11 +448,96 @@ JCL      - a786f96b13 based on jdk-11+21)
 
 ----------------------------------
 
-## MacOS
+## macOS
 :apple:
+The following instructions guide you through the process of building a macOS **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for macOS 10.
 
-:construction:
-We haven't created a full build process for macOS yet? Watch this space!
+### 1. Prepare your system
+:apple:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Xcode 4.6.2](https://xcodereleases.com/), which should be installed into the directory `/Applications/Xcode4/`.
```

Comment:
AFAIK, Xcode 4.6.2 is necessary only for OpenJDK8 builds. JDK9 and up can be build using the latest Xcode versions (9.x, 10.x), which are very easy to set up. Downloading and setting up Xcode 4.6.2 is not as straightforward, and in my experience, does not seem to work with macOS versions newer than macOS 10.11 El Capitan.

Reply:
@nbhuiyan  @DanHeidinga Which version do we want to quote in here for OpenJDK 11? I see Xcode 7 mentioned in #36?

Reply:
@SueChaplain #36 is a bit of a mess as it's the ongoing investigation into building JDK8 mixed into the JDK11 builds.

For JDK11, we should match the OpenJDK suggested build environment of `xcode 9.4` and `10.13.5 (Yosemite)` OS version.  See https://wiki.openjdk.java.net/display/Build/Supported+Build+Platforms

As this will ensure that the OpenJDK code compiles successfully as well.

Reply:
Changing to xcode 9.4. Do we need to tell them where to install xcode? If so, presumably this is different for v9.4

>..  which should be installed into the directory `/Applications/Xcode4/`.

Reply:
We don't need to tell them where to install it.  The default location is fine.

Reply:
For now I changed this to /Applications/Xcode9/ .... so please let me know if it needs further adjustment (or removing).

Reply:
OK. Removed the directory. Thanks